basetest.pm

```diff
@@ -367,6 +368,27 @@ sub next_resultname {
     }
 }
 
+=head2 record_resultfile
+
+    $self->record_resultfile($title, $output [, result => $result] [, resultname => $name]);
+
+Record result file to be parsed when evaluating test results, for example
+within the openQA web interface.
+=cut
+sub record_resultfile {
+    my ($self, $title, $output, %nargs) = @_;
+    my $filename = $self->next_resultname('txt', $nargs{resultname});
```

Comment:
yes, it's https://progress.opensuse.org/issues/10704

I am convinced we should be able to prevent *all* warnings in the tests we have.